### PR TITLE
added tpv order

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/payments/third-party-validation/#step-2-create-an-order

### CODE REVIEW

File: documents/order.md [hunk 1]
```diff
@@ -52,6 +52,55 @@ Razorpay::Order.create(para_attr)
...
+  "amount": 500,
+  "method": "netbanking",
+  "receipt": "BILL13375649",
+  "currency": "INR",
+  "bank_account": {
+    "account_number": "765432123456789",
+    "name": "Gaurav Kumar",
+    "ifsc": "HDFC0000053"
+  }
+)
```
Comment (sofiya02) [inline documents/order.md:68]: 1. undefined local variable or method **para_attr**
 2. unexpected ')', expecting '}'

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/order.md (hunk 1 above)
Comment (ankitdas13) [reply documents/order.md]: > 1. undefined local variable or method **para_attr**
> 2. unexpected ')', expecting '}'

[added](https://github.com/razorpay/razorpay-ruby/blob/third_party_validation/documents/order.md?plain=1#L58)

File: documents/order.md [hunk 2]
```diff
@@ -52,6 +52,55 @@ Razorpay::Order.create(para_attr)
...
+  "amount": 500,
+  "method": "netbanking",
+  "receipt": "BILL13375649",
+  "currency": "INR",
+  "bank_account": {
+    "account_number": "765432123456789",
+    "name": "Gaurav Kumar",
+    "ifsc": "HDFC0000053"
+  }
+
```
Comment (sofiya02) [inline documents/order.md:68]: } closing bracket missing

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/order.md (hunk 2 above)
Comment (ankitdas13) [reply documents/order.md:69]: [updated](https://github.com/razorpay/razorpay-ruby/blob/third_party_validation/documents/order.md?plain=1#L68)

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**